vyxal/elements.py

```diff
@@ -2379,6 +2379,24 @@ def matrix_determinant(lhs, ctx):
     return sympy.det(sympy.Matrix(lhs))
 
 
+def matrix_exponentiation(lhs, rhs, ctx):
+    """Element Þe
+    (lst, num) -> (a * a) b times
+    (num, lst) -> (b * b) a times
+    """
+
+    ts = vy_type(lhs, rhs, simple=True)
+    if set(ts) != {list, NUMBER_TYPE}:
+        raise TypeError("matrix exponentiation only works on lists and numbers")
+
+    matrix, times = lhs, rhs if ts[0] == NUMBER_TYPE else rhs
+    original_matrix = deep_copy(matrix)
```

Comment:
There's no need to copy, is there? `multiply` doesn't mutate its arguments

Reply:
Sure, it doesn't. But assignment does.

Reply:
@Lyxal It doesn't, `original_matrix = matrix` is enough